Oanda/modules/trader/interpreter/interpreter.py

```diff
@@ -220,9 +228,20 @@ def trade(self, prediction, latest_value):
         filter_dict(data)
         
         print(readable_output(data))
+        
         try:
-            OrdersOrderCreate(self.access_token, self.accountID, data=data)
-            print("Bought ", units, " ", self.instrument, " value of trade: ", units*latest_value)
+            for i in range(tries):
+                if i < tries:
+                    raise MissingRepsonseException
+                if self.lastTransactionID == AccountDetails(self.access_token, self.accountID)[1]['lastTransactionID']:
```

Comment:
What does this check?

Reply:
It checks if the lasttransactionID which is saved before attempting a trade is the same as the lasttransactionID from the Oanda server after attempting a trade. If it's the same, the attempted trade did not come through. If it's not the same, it did